prime-angular-frontend/src/app/modules/enrolment/pages/care-setting/care-setting.component.ts

```diff
@@ -130,6 +163,13 @@ export class CareSettingComponent extends BaseEnrolmentProfilePage implements On
     return this.careSettingTypes;
   }
 
+
+  public hasSelectedHACareSetting(): boolean {
+    // When code is in template, get: `Parser Error: Bindings cannot contain assignments`
```

Comment:
Don't know if this comment provides something we want to document since this is just something that Angular does not allow.

Reply:
I don't understand this error message as I'm not doing any assignment?  This doesn't fully explain it:  https://stackoverflow.com/questions/43117917/angular-2-bindings-cannot-contain-assignments